jupyros/__init__.py

```diff
@@ -35,7 +36,11 @@
     from .ros1.server_extension import *
     from .ros1.turtle_sim import *
 
-
+if(rclpy.ok() == True):
```

Comment:
If the environment only has ROS1, `rclpy` won't be available. It will be better to only check when the environment has ROS2 installed, or omit this check altogether.